src/applications/vaos/components/StatusAlert.jsx

```diff
@@ -43,62 +44,139 @@ export default function StatusAlert({ appointment, facility }) {
   const showConfirmMsg = queryParams.get('confirmMsg');
 
   const canceled = appointment.status === APPOINTMENT_STATUS.cancelled;
-  const { isPastAppointment } = appointment.vaos;
-  const avsLink = appointment.avsPath;
-  const avsError = avsLink?.includes('Error');
-  const displayAvsLink = () => {
-    if (avsError) {
+  const { isPastAppointment, isPendingAppointment } = appointment.vaos;
+
+  const canceler = new Map([
+    [CANCELLATION_REASONS.patient, 'You'],
+    [CANCELLATION_REASONS.provider, `${facility?.name || 'Facility'}`],
+  ]);
+  const { status } = appointment;
+
+  if (
+    featureAfterVisitSummary &&
+    featureAppointmentDetailsRedesign &&
+    APPOINTMENT_STATUS.proposed === status
+  ) {
+    if (isPastAppointment) {
       return (
-        <div
-          aria-atomic="true"
-          aria-live="assertive"
-          className="vads-u-margin-top--2 vads-u-margin-bottom--0"
-        >
-          <InfoAlert
-            status="error"
-            level={1}
-            headline="We can't access after-visit summaries at this time."
-          >
-            We’re sorry. We’ve run into a problem.
-          </InfoAlert>
+        <InfoAlert backgroundOnly status="info">
+          <p>
+            We’ll try to schedule your appointment in the next 2 business days.
+            Check back here or call your facility for updates. Review your
+            appointments Schedule a new appointment
+          </p>
```

Comment:
The last sentence "Review your appointments Schedule a new appointment" can be removed.

Reply:
Done.